packages/nx-semantic-release/src/tests/setup-test-repo.ts

```diff
@@ -105,12 +105,12 @@ function createPackageJsonForProjects() {
 async function bootstrapTestProjectsAndLibs() {
   testApps.forEach((project) => {
     runNxCommand(
-      `generate @nx/web:application ${project} --e2e-test-runner=none`
+      `generate @nx/web:application ${project} --directory apps/${project} --e2e-test-runner=none`
```

Comment:
Seems like the preferred way to use these generators is now to provide explicitly the directory where to generate

![image](https://github.com/TheUnderScorer/nx-semantic-release/assets/3244050/016be3f5-e977-4816-8fb3-e5eb45b7444d)

https://nx.dev/nx-api/web/generators/application

Reply:
Seems like when upgrading Nx next time we might need to change this again. In 16.9.0 is seemed to be just `apps/` for some reason